next.config.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
     HUMAN_ID_GET_WEB_LOGIN_URL: process.env.HUMAN_ID_GET_WEB_LOGIN_URL,
     HUMAN_ID_CLIENT_ID: process.env.HUMAN_ID_CLIENT_ID,
     HUMAN_ID_CLIENT_SECRET: process.env.HUMAN_ID_CLIENT_SECRET,
+    REDIRECT_MAIN_URL: process.env.REDIRECT_MAIN_URL || '',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `REDIRECT_MAIN_URL` environment variable is given a default value of an empty string. This could potentially lead to unexpected behavior if the environment variable is not set and the code attempts to use it for redirection. It would be better to provide a more sensible default or handle the case where the URL might be an empty string.
```diff
- REDIRECT_MAIN_URL: process.env.REDIRECT_MAIN_URL || '',
+ REDIRECT_MAIN_URL: process.env.REDIRECT_MAIN_URL || 'https://defaulturl.com',
```
Alternatively, you can add error handling in the code that uses this variable to ensure it's not an empty string before using it.


<!-- This is an auto-generated comment by OSS CodeRabbit -->